src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -217,12 +217,12 @@ internal ImmutableArray<MethodSymbol> GetOperators(string name)
                 return ImmutableArray<MethodSymbol>.Empty;
             }
 
-            ArrayBuilder<MethodSymbol> operators = ArrayBuilder<MethodSymbol>.GetInstance();
-            foreach (MethodSymbol candidate in candidates.OfType<MethodSymbol>())
+            var operators = ArrayBuilder<MethodSymbol>.GetInstance(candidates.Length);
```

Comment:
passing in capacity is appropriate here.  Callers are always passing operator names.  So the *normal* case is that all the members we found with that name *are* going to actually be operators, and thus would be added in the loop below.